### PR TITLE
Nerf Marine Armor

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
Too much buff. we move back now. nerf. yes yes. nerf. 

## Why It's Good For The Game
I am the strongest balance warrior 
## Changelog

:cl:
balance: marine armor is now less protective, marginally, and slows you down |more|
del: Marine Helmets
/:cl:

### CODE REVIEW

**Comment by Burning02** [comment]
I think Sudie probably meant remove the helmet from the crate, not the code?

**Comment by Erikafox** [comment]
i read it as 'the sprite is ugly and i think it can go' but if that's not correct... easy fix

**Comment by thgvr** [comment]
> I think Sudie probably meant remove the helmet from the crate, not the code?

It's unused at that point, why keep it?

File: code/modules/cargo/packs/spacesuit_armor.dm [hunk 1]
```diff
@@ -150,6 +150,6 @@
 	desc = "One set of well-rounded medium tactical body armor. The set includes a helmet and vest."
 	cost = 2000
```
Comment (thgvr) [inline code/modules/cargo/packs/spacesuit_armor.dm:151]:
```suggestion
	cost = 5000
```

File: code/modules/cargo/packs/spacesuit_armor.dm [hunk 2]
```diff
@@ -141,7 +141,7 @@
 	desc = "One set of well-rounded, tactical body armor. The set includes a helmet and vest."
 	cost = 1500
```
Comment (thgvr) [inline code/modules/cargo/packs/spacesuit_armor.dm:142]:
```suggestion
	cost = 3000
```

**Review by thgvr — DISMISSED** [review]

File: code/modules/cargo/packs/spacesuit_armor.dm (hunk 2 above)
Comment (Erikafox) [reply code/modules/cargo/packs/spacesuit_armor.dm:142]: i think 3000 is too high for an armored vest upgrade (750)

Comment (Erikafox) [reply code/modules/cargo/packs/spacesuit_armor.dm:142]: also im not doing that in this pr because im doing it in #3781

**Pull request merged**